finder-app/Makefile

```diff
@@ -0,0 +1,36 @@
+#template taken from https://stackoverflow.com/questions/1484817/how-do-i-make-a-simple-makefile-for-gcc-on-linux/1484873#1484873
+INCLUDE_DIRS = #-I/usr/include/opencv4
+LIB_DIRS = 
+CROSS_COMPILE = #aarch64-none-linux-gnu
+TARGET = writer
+CC=$(CROSS_COMPILE)-gcc
```

Comment:
@ceca5556 note that the CROSS_COMPILE definition should include a trailing `-`, so you will have one extra `-` here.